app/models/state_file_nc_intake.rb

```diff
@@ -38,8 +38,12 @@
 #  primary_middle_initial            :string
 #  primary_suffix                    :string
 #  raw_direct_file_data              :text
+#  raw_direct_file_intake_data       :jsonb
+#  raw_direct_file_tax_return_data   :text
 #  referrer                          :string
 #  routing_number                    :integer
+#  sales_use_tax                     :decimal(12, 2)
+#  sales_use_tax_calculation_method  :integer          default(0), not null
```

Comment:
why are these added in this PR? also for below `untaxed_out_of_state_purchases`

Reply:
they're just there because i didn't roll back the migration from my work on that branch before starting work on this branch. this diff should disappear once that branch is merged